### PR TITLE
Adding NAMD3 files for ROCm-4.2.0

### PR DESCRIPTION
**Pull request by sanjtrip**
Adding NAMD-3 files for ROCm-4.2.0

### CODE REVIEW

File: namd/rocm420/ubuntu18/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+sudo docker run -it --privileged --ipc=host --network=host --device=/dev/kfd --device=/dev/dri --group-add video --cap-add=SYS_PTRACE --security-opt seccomp=unconfined amddcgpuce/namd3-rocm420-ubuntu18 bash
+```
+#### On CentOS/RHEL 8.x, use podman:
+```
+# Launch container in interactive mode, bash shell
+sudo podman run -it --privileged docker://amddcgpuce/namd3-rocm420-ubuntu18 bash
+```
+##### Image Name + Tag | Description
+```
+    namd3-rocm420-ubuntu18:version1   -->   NAMD-3 based on ROCm v4.2.0 On Ubuntu18 (Version 1)
```
Comment (amddcgpuce) [inline namd/rocm420/ubuntu18/README.md:16]: Is `version1` the tag?

**Review by amddcgpuce — COMMENTED** [review]
Please check docker image tagname

File: namd/rocm420/ubuntu18/README.md (hunk 1 above)
Comment (sanjtrip) [reply namd/rocm420/ubuntu18/README.md:16]: Yes !

Comment (amddcgpuce) [reply namd/rocm420/ubuntu18/README.md:16]: > Yes !

@sanjtrip Will you be fixing it?

Comment (sanjtrip) [reply namd/rocm420/ubuntu18/README.md:16]: Any input on possible alternative like the following ? 

namd3-rocm420-ubuntu18:version1   -->   NAMD-3 based on ROCm v4.2.0 On Ubuntu18 (Tag: version1)

Comment (amddcgpuce) [reply namd/rocm420/ubuntu18/README.md:16]: Let's close this PR so you can figure it out and resubmit it. As it is, this is an error.

Comment (sanjtrip) [reply namd/rocm420/ubuntu18/README.md:16]: Sure ! I will resubmit after fixing it.

File: namd/rocm420/ubuntu18/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+sudo docker run -it --privileged --ipc=host --network=host --device=/dev/kfd --device=/dev/dri --group-add video --cap-add=SYS_PTRACE --security-opt seccomp=unconfined amddcgpuce/namd3-rocm420-ubuntu18 bash
+```
+#### On CentOS/RHEL 8.x, use podman:
+```
+# Launch container in interactive mode, bash shell
+sudo podman run -it --privileged docker://amddcgpuce/namd3-rocm420-ubuntu18 bash
+```
+##### Image Name + Tag | Description
+```
+    namd3-rocm420-ubuntu18:version1   -->   NAMD-3 based on ROCm v4.2.0 On Ubuntu18 (Tag : version1)
```
Comment (amddcgpuce) [inline namd/rocm420/ubuntu18/README.md:16]: The `version` tag for docker image is nonexistent. Line 16 is still incorrect.

**Review by amddcgpuce — COMMENTED** [review]
The `version` docker tagname is nonexistent, should be removed.

File: namd/rocm420/ubuntu18/README.md (hunk 2 above)
Comment (sanjtrip) [reply namd/rocm420/ubuntu18/README.md]: Removed the tag - updated line#16.

**Pull request merged**